internal/fsutil/create.go

```diff
@@ -67,5 +67,13 @@ func createSymlink(o *CreateOptions) error {
 	if err != nil && !os.IsExist(err) {
 		return err
 	}
+	// check if symlink exists, remove if it does
+	_, err = os.Lstat(o.Path)
+	if err == nil {
+		err = os.Remove(o.Path)
```

Comment:
It'd be better to first check that `os.Readlink(o.Path) != o.Path` before removing the symlink and writing the new one.

Reply:
Correction: `os.Readlink(o.Path) != o.Link` ... or the other way. :-)

Reply:
Yeah, this needs to be fixed at the place where the symlink is actually written. I guess you can drop this hunk and try to work from here https://github.com/woky/chisel/blob/main/internal/slicer/slicer.go#L130 and here https://github.com/woky/chisel/blob/main/internal/fsutil/create.go#L18.